charts/firefly-signer/values.yaml

```diff
@@ -0,0 +1,15 @@
+image:
+  repository: ghcr.io/hyperledger/firefly-signer
+  pullPolicy: IfNotPresent
+  # tag: ""
```

Comment:
1.3 tag? feels weird to have IfNotPresent with latest

Reply:
https://github.com/hyperledger/firefly-helm-charts/pull/76/commits/205f20754d1f00a68130ca2234e50bf3db86aca8